test/representers/api/paged_collection_representer_test.rb

```diff
@@ -37,7 +37,7 @@
     define_routes
     representer.represented.options[:parent] = TestParent.new(1, true)
     representer.represented.options[:item_class] = TestObject
-    representer.href_url_helper({page: 1}).must_equal "/api/test_parent/1/test_objects?page=1"
+    representer.href_url_helper({page: 1}).must_equal "/api/test_parents/1/test_objects?page=1"
```

Comment:
Line is too long. [95/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.